connector-template/mod.rs

```diff
@@ -551,3 +553,24 @@ impl webhooks::IncomingWebhook for {{project-name | downcase | pascal_case}} {
         Err(report!(errors::ConnectorError::WebhooksNotImplemented))
     }
 }
+
+impl ConnectorSpecifications for {{project-name | downcase | pascal_case}} {
+    fn get_connector_about(&self) -> Option<ConnectorInfo> {
+        Some(ConnectorInfo {
+            description:
+                "Connector About"
+                    .to_string(),
+            connector_type: PaymentConnectorCategory::PaymentGateway,
+        })
+    }
+
+    fn get_supported_payment_methods(&self) -> Option<SupportedPaymentMethods> {
+        let mut supported_payment_methods  = SupportedPaymentMethods::new();
```

Comment:
Try to use const hashmaps here instead of constructing the hashmap during runtime, since we already have this information during compile time, we can construct the config